plugins/rest_connector/api/tasks.py

```diff
@@ -79,9 +82,27 @@ def perform_request(self, connector_id: str, db_id: int) -> None:
         timeout=20,  # TODO allow different timeouts?
     )
 
-    # FIXME response handling...
+    template_context: Dict[str, Any] = request_variables | {
+        "request": response.request,
+        "response": response.json(),
+    }
+
+    for response_map in response_mapping:
+        with SpooledTemporaryFile(mode="w") as output:
+            content = render_template_sandboxed(response_map["data"], template_context)
```

Comment:
TODO: files that have the "dereference urls" option set will have a url in the content that we need to dereference. (we could use the URL File store for this.)